workers/paymaster/tests/index.test.ts

```diff
@@ -397,6 +410,53 @@ describe('PaymasterWorker', () => {
     });
   });
 
+  test('Should not send notification if paidUntil is set to the several months in the future', async () => {
```

Comment:
please add a test case: "should send notification if payday is coming for workspace with paidUntil value"